packages/taquito-sapling/src/sapling-keys/in-memory-spending-key.ts

```diff
@@ -49,14 +49,14 @@ export class InMemorySpendingKey {
 
   /**
    *
-   * @param mnemonic list of words
+   * @param mnemonic strings of words
    * @param derivationPath tezos current standard 'm/'
    * @returns InMemorySpendingKey class instantiated
    */
 
-  static async fromMnemonic(mnemonic: string[], derivationPath = 'm/') {
+  static async fromMnemonic(mnemonic: string, derivationPath = 'm/') {
     // no password passed here. password provided only changes from sask -> MMXj
-    const fullSeed = await bip39.mnemonicToSeed(mnemonic.join(' '));
+    const fullSeed = await bip39.mnemonicToSeed(mnemonic);
```

Comment:
I replaced the array for a string for consistency with the `InMemorySigner` where we also use a string. 
Also, `bip39.generateMnemonic()` returns a string, so I found it more convenient to accept a string in the method.
Yes, if there is any errors in spacing it could cause a problem